### PR TITLE
On-ramp: Add same amount rendering as the order details to the order list

### PR DESCRIPTION
**Pull request by wachunei**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

Ensure consistent crypto number formatting across orders list and order details view.

**Screenshots/Recordings**

~~_If applicable, add screenshots and/or recordings to visualize the before and after of your change_~~

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/components/Views/FiatOrdersView/OrderListItem.js [hunk 1]
```diff
@@ -31,6 +33,23 @@ const styles = StyleSheet.create({
  * @param {FiatOrder} props.order
  */
 function OrderListItem({ order }) {
+  let amount = '...';
+  if (order.cryptoAmount) {
+    if (
```
Comment (sethkfman) [inline app/components/Views/FiatOrdersView/OrderListItem.js:38]: I would break this out in to a util function.

Comment (wachunei) [reply app/components/Views/FiatOrdersView/OrderListItem.js]: Addressed in [15a4bcb](https://github.com/MetaMask/metamask-mobile/pull/6139/commits/15a4bcba8a5900de6db01591aa82a4f0178d790a)

File: app/components/UI/FiatOnRampAggregator/utils/index.test.ts [hunk 2]
```diff
@@ -67,3 +76,73 @@ describe('formatId', () => {
...
+
+  it('should render correctly', () => {
+    expect(
+      getOrderAmount({
+        ...mockOrder,
+        cryptoAmount: 0.012361263,
+      }),
+    ).toBe('0.01236');
+
+    expect(
```
Comment (Cal-L) [inline app/components/UI/FiatOnRampAggregator/utils/index.test.ts:137]: Can we split this into its own test and add more description to the title - For example: `should render correctly when data is provided`

**Comment by Cal-L** [comment]
Could you provide visual snapshots of the changes where this is affected?

File: app/components/UI/FiatOnRampAggregator/utils/index.test.ts [hunk 3]
```diff
@@ -67,3 +76,73 @@ describe('formatId', () => {
...
+      getOrderAmount({
+        ...mockOrder,
+        cryptoAmount: 0.012361263,
+        data: {
+          ...mockOrder.data,
+          decimals: undefined,
+        } as Order,
+      }),
+    ).toBe('0.01236');
+  });
```
[the remark this answers is not in the document]
Comment (wachunei) [reply app/components/UI/FiatOnRampAggregator/utils/index.test.ts:147]: It can't be `undefined`

**Comment by wachunei** [comment]
image

These display different ETH amount, they will render the same after this change.

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**